server/src/db.py

```diff
@@ -0,0 +1,57 @@
+import os
+from typing import Any, Dict, List, Optional
```

Comment:
let's add a todo here somewhere that we repeat exactly this code in indexer, and we should do something about it in the future.

just a plain: # todo: blah blah

comment somewhere here and in the other db file, please